addons/sale_project/tests/test_project_profitability.py

```diff
@@ -96,12 +153,12 @@ def test_project_profitability(self):
                     ],
                     'total': {
                         'to_invoice': self.delivery_service_order_line.untaxed_amount_to_invoice,
-                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced,
+                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced+100,
```

Comment:
You can let the `+` breathing a little bit. 😇 
```suggestion
                        'invoiced': self.delivery_service_order_line.untaxed_amount_invoiced + 100,
```